server/chat-plugins/datasearch.ts

```diff
@@ -1211,6 +1211,7 @@ function runMovesearch(target: string, cmd: string, canAll: boolean, message: st
 			if (target === 'z') target = 'zmove';
 			if (target === 'max') target = 'maxmove';
 			if (target === 'gmax') target = 'gmaxmove';
+			if (target === 'crit' || toID(target) === 'highcrit') target = 'highcrit';
 			if (allFlags.includes(target)) {
 				if ((orGroup.flags[target] && isNotSearch) || (orGroup.flags[target] === false && !isNotSearch)) {
 					return {error: 'A search cannot both exclude and include \'' + target + '\'.'};
```

Comment:
```suggestion
					return {error: `A search cannot both exclude and include '${target}'.`};
```
While you're refactoring stuff, this is more readable imo.